packages/samples/test/output/grpc-kiosk-example/openapi.json

```diff
@@ -404,11 +404,19 @@
             "description": "name of device hardware"
           },
           "size": {
-            "$ref": "#/components/schemas/ScreenSize",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/ScreenSize"
+              }
+            ],
             "description": "screen size"
           },
           "location": {
-            "$ref": "#/components/schemas/LatLng",
+            "allOf": [
```

Comment:
This is unfortunate looking, but you're right, OpenAPI spec doesn't allow what we had before so +1. Weirdly https://editor.swagger.io/ tolerates it too.

I kind of wish this were formatted `"allOf": [{ "$ref": "#/components/schemas/ScreenSize" }]` on line noise, this is a lot of noise to add a description to a property. :(

Reply:
well it makes it a warning 
![image](https://user-images.githubusercontent.com/1031227/181596128-8bd2f31b-c4b3-4989-bac2-6b436658674f.png)

yeah agree with the formatting not sure if we can easily solve that though

Reply:
Hmm, I missed the warning. That's reassuring.